addons/l10n_sa/account.chart.template.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+
+   <record id="account_arabic_coa_general" model="account.chart.template">
+            <field name="name">General Arabic CoA</field>
+            <field name="spoken_languages" eval="'en_US;ar_EG;ar_SY'"/>
+            <field name="account_root_id" ref="coa_basic"/>
+            <field name="complete_tax_set" eval="False" />
+			<field name="tax_code_root_id" ref="account.tax_code_chart_root"/>  
```

Comment:
Check your indentation, you are mixing spaces and tabs. Use only one in your files (preferably spaces)